libr/anal/dwarf_process.c

```diff
@@ -57,6 +57,7 @@ static void variable_free(Variable *var) {
 	free (var);
 }
 
+#if 0
```

Comment:
remove if not needed

Reply:
Well im not touching the dwarf code but this apits out a warning because unused. I can remove but i would prefer to get the feedback from the one who wrote this (in the phone now, no git blame handy)

Reply:
@HoundThe ok to remove this?

Reply:
Yes, it is